### ISSUE TITLE
PyData Sphinx - Review the design system for focus state - buttons, dropdowns

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/czi-scientific-python-mgmt#33]
## Acceptance Criteria

```[tasklist]
### Tasks
- [ ] https://github.com/Quansight-Labs/czi-scientific-python-mgmt/issues/36
- [ ] https://github.com/Quansight-Labs/czi-scientific-python-mgmt/issues/35
- [x] Navigation components (navbar) meet criteria
- [x] Tab component meets criteria
```

## Details (criteria)

1. Text contrast: should be met now after updating the colours WCAG SC 1.4.3 - text contrast AA](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum)
	- 4.5:1 ratio for text and background (text < 18pt or bold text < 14pt)
	- 3:1 ratio for text and background (text >= 18pt or bold text =< 14pt)
2.  Focus visible criteria - buttons, links, and navigation components meet at least WCAG 2.1 (designs account for changes in [WCAG 2.2](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)
	-  [WCAG 2.1 SC 2.4.7](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible)
3. [WCAG SC 1.4.11 - non-text contrast AA](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast) need a minimum 3:1 contrast ratio with adjacent colours
    - links should meet this already
    - need to double check for `focus visible`
4.  [WCAG SC 1.4.1 - use of colour](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color)
	- Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing an optical element
5. [WCAG SC 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)
6. 

## Resources

- [Sara S. post on focus indicators](https://www.sarasoueidan.com/blog/focus-indicators/)
- [theme-components: include tabs and others](https://pydata-sphinx-theme--1376.org.readthedocs.build/en/1376/user_guide/web-components.html)

**Comment by smeragoel** [comment]
Sounds good, I'll get started on this

**Comment by trallard** [comment]
Relevant: https://github.com/pydata/pydata-sphinx-theme/issues/1387
